rust/main/agents/relayer/src/msg/metadata/message_builder.rs

```diff
@@ -392,8 +392,7 @@ mod test {
         let params = MessageMetadataBuildParams::default();
         let err = build_message_metadata(message_builder, ism_address, &message, params.clone())
             .await
-            .err()
-            .expect("Metadata found when it should have failed");
+            .expect_err("Metadata found when it should have failed");
```

Comment:
nice, I didn't realize this was a thing xd

Reply:
me neither lol, this was clippy's doing